build/stf-run-ci/templates/manifest_elasticsearch.j2

```diff
@@ -0,0 +1,52 @@
+apiVersion: elasticsearch.k8s.elastic.co/v1
```

Comment:
Copy/trim of https://github.com/infrawatch/service-telemetry-operator/blob/master/roles/servicetelemetry/templates/manifest_elasticsearch.j2

Reply:
This will be good information for the KCS after forwarding becomes the supported interface